apps/storefront/src/utils/b3DateFormat/index.ts

```diff
@@ -38,7 +41,7 @@ const formatCreator =
       case 'parseDate':
         return fmt.parseDate(dateObject, display) || '';
       default:
-        return null;
+        throw new Error('Invalid value');
```

Comment:
This should never happen, the value of `handler` is protected by TS.

This removes `null` from the output of this method, removing some of the instances of the translated value being `null`.